.github/workflows/maven.yml

```diff
@@ -5,7 +5,6 @@ name: Java CI with Maven
 
 on:
   push:
```

Comment:
is this still needed?

Reply:
I need actions to work on every branch in my fork. It doesn't change anything for upstream as long as it only has a single master branch.